flash/core/data/data_source.py

```diff
@@ -697,3 +709,218 @@ def _get_classes(self, data):
             classes = data.distinct(label_path)
 
         return classes
+
+
+class LabelStudioDataSource(DataSource):
+    """The ``LabelStudioDatasource`` expects the input to
+    :meth:`~flash.core.data.data_source.DataSource.load_data` to be a json export from label studio."""
+
+    def __init__(self):
+        super().__init__()
+        self.results = []
+        self.test_results = []
+        self.val_results = []
+        self.classes = set()
+        self.data_types = set()
+        self.num_classes = 0
+        self._data_folder = ""
+        self._raw_data = {}
+        self.multi_label = False
+        self.split = 0.8
+
+    def load_data(self, data: Optional[Any] = None, dataset: Optional[Any] = None) -> Sequence[Mapping[str, Any]]:
+        """Iterate through all tasks in exported data and construct train\test\val results."""
+        if data and isinstance(data, dict):
+            self._data_folder = data.get("data_folder")
+            with open(data.get("export_json")) as f:
+                self._raw_data = json.load(f)
+            self.multi_label = data.get("multi_label", False)
+            self.split = data.get("split", 0.8)
+        for task in self._raw_data:
```

Comment:
Could this code be unit-tested and moved to utility function?
Is it common to all tasks supported by LabelStudio ?

Reply:
It's a common step for all tasks. 
Added unit test for this method.